docs/# Using the Inspect API to load a derefe.md

```diff
@@ -0,0 +1,35 @@
+# Using the Inspect API to load a dereferenced CSVW.
```

Comment:
I wouldn't include the `#` in the name of the file - and I'd try to avoid spaces in the name too.